package.json

```diff
@@ -253,6 +256,24 @@
 					"dark": "/resources/icons/theme-agnostic/UpdateOutline.svg"
 				}
 			},
+			{
+				"category": "CosmosDB",
+				"command": "cosmosDB.updateMongoCollection",
```

Comment:
Can this be 'Update Collection to Cloud'? The users can attach a mongo database from any cloud